.travis.yml

```diff
@@ -1,6 +1,6 @@
 branches:
   only:
-    - master
+    - /v\d+\.\d+\.\d+/
```

Comment:
This needs to match to a tag name, not a branch name.

Reply:
Based on https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches I believe having a regex in branches.only will only build tagged commits.

Tagged commit build: https://travis-ci.com/github/janaknat/gensim/builds/214207365